src/util.cpp

```diff
@@ -198,6 +198,11 @@ boost::filesystem::path GetDebugLogPath()
     return GetDataDir() / "debug.log";
 }
 
+boost::filesystem::path GetMCCryptoLogPath()
+{
+    return GetDataDir(false) / "mc_crypto.log";
```

Comment:
Why is the parameter `fNetSpecific` set to `false`? I see an inconstistency with `GetDebugLogPath` (everything would be written as mainnet).

Reply:
I was also in doubt on this, but in the end I preferred this solution for simplicity. Having network specific paths would involve having 3 different configuration files, which must be copied (and possibly configured) on each first boot on given networks.
Given the amount of data logged (and its consistent format), I thought that it was not worth splitting the logs in network specific log files, and preferred logging in the main datadir only.

Reply:
Ok, your explanation makes sense.
Maybe we could put a very brief comment just to make clear this was a decision we are aware of and not an oversight. But it's just up to you, I'm fine with your reply.